<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardsWidget</class>
 <widget class="QFrame" name="CardsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.559, y1:0, x2:0.598, y2:1, stop:0.335196 rgba(137, 207, 240, 255), stop:1 rgba(209, 209, 209, 255));</string>
  </property>
  <widget class="QFrame" name="nowFrame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelNow">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>&lt;holder&gt; Now</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iconLabel">
        <property name="text">
         <string>&lt;holder&gt; image</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNowValue">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>&lt;valuePlaceHolder&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>51</width>
     <height>251</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>183</red>
         <green>208</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>64</red>
         <green>65</green>
         <blue>66</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>183</red>
         <green>208</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.559000000000000" starty="0.000000000000000" endx="0.598000000000000" endy="1.000000000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.335196000000000">
          <color alpha="255">
           <red>137</red>
           <green>207</green>
           <blue>240</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>209</red>
           <green>209</green>
           <blue>209</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QFrame" name="predictedFrame">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>20</y>
     <width>251</width>
     <height>221</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="labelPredicted">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>255</green>
              <blue>255</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="HighlightedText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>240</red>
              <green>240</green>
              <blue>240</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="text">
         <string>&lt;holder&gt; Predicted</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="emilWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>245</width>
          <height>260</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
